### PR TITLE
Fix annotation card scroll

### PR DESCRIPTION
**Pull request by brmscheiner**
Addresses DEX-883

### CODE REVIEW

File: src/pages/sighting/encounters/AnnotationsCard.jsx [hunk 1]
```diff
@@ -61,54 +61,56 @@ export default function AnnotationsCard({
...
-                    })
-                  }
-                  style={{
-                    position: 'absolute',
-                    top: 12,
-                    right: 12,
-                    color: theme.palette.common.white,
-                  }}
-                  className={classes.photoIcon}
+        <div style={{ margin: '8px 0' }}>
```
Comment (brmscheiner) [inline src/pages/sighting/encounters/AnnotationsCard.jsx:64]: surrounding the Grid in a div with 8px margin on top and bottom cancels out the -8px from material-ui

**Review by Atticus29 — APPROVED** [review]
Good to know. Thanks!

File: src/pages/sighting/encounters/AnnotationsCard.jsx (hunk 1 above)
Comment (Emily-Ke) [reply src/pages/sighting/encounters/AnnotationsCard.jsx:64]: I think the style could have been added directly to the Grid and saved you one div.

**Review by Emily-Ke — APPROVED** [review]

File: src/pages/sighting/encounters/AnnotationsCard.jsx (hunk 1 above)
Comment (brmscheiner) [reply src/pages/sighting/encounters/AnnotationsCard.jsx]: that works! changed

**Pull request merged**